Cargo.toml

```diff
@@ -45,58 +49,61 @@ payjoin = { version = "0.6.0", features = ["sender"] }
 pretty_env_logger = "0.4.0"
 regex = "1.7.0"
 reqwest = { version = "0.11.13", features = ["json"] }
+rgb-std = { version = "0.10.0-rc.1" }
+rgb-wallet = {version = "0.10.0-beta.1" }
 serde = "1.0.152"
 serde_json = "1.0.91"
 serde-encrypt = "0.7.0"
-tokio = { version = "1.23.0", features = ["macros", "sync"] }
+strict_types = "1.0.0-rc.1"
+tokio = { version = "1.27.0", features = ["macros", "sync"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 bdk = { version = "0.27.1", features = [
     "use-esplora-async",
     "async-interface",
 ], default-features = false }
 gloo-console = "0.2.3"
-gloo-net = "0.2.5"
-js-sys = "0.3.60"
-serde-wasm-bindgen = "0.4.5"
-wasm-bindgen = { version = "0.2.83", features = ["serde-serialize"] }
-wasm-bindgen-futures = "0.4.33"
-web-sys = "0.3.60"
+gloo-net = "0.2.6"
+js-sys = "0.3.61"
+serde-wasm-bindgen = "0.5.0"
+wasm-bindgen = { version = "0.2.84", features = ["serde-serialize"] }
+wasm-bindgen-futures = "0.4.34"
+web-sys = "0.3.61"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-amplify = "3.13.0"
+amplify_legacy = { package = "amplify", version = "3.13.0" }
```

Comment:
Is there any reason to do legacy here? We should have the same functionality either way, no?

Reply:
Yes, amplify v4 was released!

The major milestone is a full compatibility with wasm. 

My goal is remove all *_legacy crates after finish the work!